tasks/add_new_openwrap_partner.py

```diff
@@ -639,6 +727,90 @@ def get_creative_file(setup_type):
 
     return creative_file
 
+def create_deal_line_item_configs(deal_config, order_id, placement_ids, bidder_code, sizes, key_gen_obj,
+  lineitem_type, lineitem_prefix, currency_code, custom_targeting, setup_type, ad_unit_ids=None, same_adv_exception=False, 
+  device_category_ids=None,device_capability_ids=None, roadblock_type='ONE_OR_MORE', durations = None, slot = None):
+  """
+  Create a line item config for each dealtier.
+
+  Args:
+    prices (array)
+    order_id (int)
+    placement_ids (arr)
+    bidder_code (str or arr)
+    sizes (arr)
+    key_gen_obj (obj)
+    lineitem_type (str)
+    lineitem_prefix (str)
+    currency_code (str)
+    custom_targeting (arr)
+    setup_type (str)
+    creative_template_ids (arr)
+    ad_unit_ids (arr)
+    same_adv_exception(bool)
+    device_category_ids (int)
+    device_capability_ids (int)
+    roadblock_type (str)
+    durations(array)
+    slot(string) 
+  Returns:
+    an array of objects: the array of DFP line item configurations
+  """
+
+ 
+  key_gen_obj.set_setup_type(setup_type)
+
+  # Set DFP targeting for custom targetting passed in settings.py
+  key_gen_obj.set_custom_targeting(custom_targeting)
+  key_gen_obj.set_creative_targeting(durations,slot)  
+
+  line_items_config = []
+     
+  #create line item config for each price
+  for bidder, cfg in deal_config.items():
+    if bidder_code is not None:
+        found = False
+        for bc in bidder_code:
+            if bidder == bc:
+                found = True
+                break
+        if found == True:
```

Comment:
@pm-isha-bharti  
We will be adding the pwtpid (bidder code) targeting only if bidderName in the deal_config is also present in the bidder_code array. 

Added the `if found == True` to set the  bidder_code targeting criteria  to none in cases where pwtpid targeting is not needed.  Earlier  bidder_code once set was used for all the lineitems.
 
Added the code to address this bug:  https://inside.pubmatic.com:9443/jira/browse/OTT-1298